tools/generator/sdk/versions.go

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+package sdk
```

Comment:
We might need to move the `autorest_ext/changelog` file to this package since it does not have any relation with `autorest` but something with the sdk

Reply:
fixed